src/core/ledger/distributionSummary/distributionSummary.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+
+import {Ledger} from "../ledger";
+import {type Distribution} from "../distribution";
+import * as G from "../grain";
+import sortBy from "../../../util/sortBy";
+import {type IdentityId} from "../../identity";
+import * as NullUtil from "../../../util/null";
+import {type CurrencyDetails} from "../../../api/currencyConfig";
+
+export function distributionMarkdownSummary(
+  distribution: Distribution,
+  ledger: Ledger,
+  currencyDetails: CurrencyDetails
+) {
+  const {name: currencyName, suffix: currencySuffix} = currencyDetails;
+  const distributionBalances: DistributionBalances = getDistributionBalances(
+    distribution
+  );
+  const totalDistributed: G.Grain = getTotalDistributed(distributionBalances);
+
+  const ids = Array.from(distributionBalances.keys());
+  const countIdentities = ids.length;
+
+  console.log(`## ${currencyName.toUpperCase()} Distribution`);
+  console.log(
+    `#### Distributed ${G.format(
+      totalDistributed,
+      0,
+      currencySuffix
+    )} to ${countIdentities} identities`
+  );
```

Comment:
just add a sum row to teh bottom for the total distributed. don't need the identity count really.

Reply:
Ack, removed identity count.  Changed line to (for example) `#### Total Distributed: 1,200g`.  I much prefer knowing the sum right upfront in the header.

Reply:
Done.